subsys/bluetooth/controller/ll_sw/ull_connections.c

```diff
@@ -453,13 +453,13 @@ void ull_conn_setup(memq_link_t *link, struct node_rx_hdr *rx)
 	switch (lll->role) {
 #if defined(CONFIG_BT_CENTRAL)
 	case 0:
-		ull_master_setup(link, rx, ftr, lll);
+		ull_master_setup(rx, ftr, lll);
```

Comment:
above - before the assignment to ftr, an assignment of link into rx is missing:
	/* Store the link in the node rx so that when done event is
	 * processed it can be used to enqueue node rx towards LL context
	 */
	rx->link = link;

Reply:
good catch. There have been multiple changes in the ull_conn.c file, so maybe time to look again at what we need to take over from there